src/System.Windows.Forms/src/System/Windows/Forms/ListViewGroup.cs

```diff
@@ -172,6 +191,29 @@ public HorizontalAlignment FooterAlignment
             }
         }
 
+        internal bool Focused { get; set; }
+
+        private bool IsDefault
+        {
+            get
+            {
+                if (ListView?.Groups == null)
+                {
+                    return false;
+                }
+
+                foreach (var group in ListView.Groups)
+                {
+                    if (group == this)
+                    {
+                        return false;
+                    }
+                }
+
+                return true;
+            }
```

Comment:
Why can't we do this?
```suggestion
        private bool IsDefault => ListView?.DefaultGroup == true;
```

Why do we need this property at all?

Reply:
Because of strange definition of this group. We have DefaultGroup object but we have no items related to this group. It always empty and can't be changed. So, to define "default" group in accessible tree we creating a new group object and set it as default via this flag and map items to it.